core/src/main/scala/cats/syntax/either.scala

```diff
@@ -372,6 +398,10 @@ final class EitherObjectOps(private val either: Either.type) extends AnyVal {
 
   def rightNel[A, B](b: B): EitherNel[A, B] = Right(b)
 
+  def leftA[F[_]: Applicative, A, B](a: A): Either[F[A], B] = Left(Applicative[F].pure(a))
+
+  def rightA[F[_], A, B](b: B): Either[F[A], B] = Right(b)
```

Comment:
To me this method seems unnecessary (similar to `validA` that Oscar pointed out previously), wdyt?

Reply:
yep, agree!